src/audiomixerboard.cpp

```diff
@@ -741,6 +742,16 @@ void CAudioMixerBoard::SetPanIsSupported()
     SetDisplayPans ( bDisplayPans );
 }
 
+void CAudioMixerBoard::SetServerRecordingState ( QString state )
+{
+    strServerRecordingState = state;
+    QString strTitle = this->title().remove ( QRegExp ( " : ~~ .* ~~" ) );
+    if ( strServerRecordingState != "" && strTitle.mid ( 0, 1 ) != "\u2588" )
+    {
+        setTitle ( strTitle + " : ~~ " + strServerRecordingState + " ~~" );
```

Comment:
These ~~ are at at least three places in the code. A single source would be preferred.

Reply:
I'd rather it wasn't done like this at all but as a separate indicator somewhere on the UI.  I'd like a little red "led" on in the top left-hand corner of the server group box, really.  But it would need to be tri-state.

Reply:
Adding the information in the title of the audio mixer board is totally fine. My complain was just about the single source problem.

Reply:
Hopefully the latest version is less intrusive yet just as informative.